website/cue/reference/components/sinks/pulsar.cue

```diff
@@ -16,6 +16,11 @@ components: sinks: pulsar: {
 		acknowledgements: true
 		healthcheck: enabled: true
 		send: {
+			batch: {
```

Comment:
You should drop the changes to this file. This is meant for configuring certain global docs.

Reply:
Fixed.

P.S. I've always thought that this file relates to the same docs... Thanks for the clarification!